docs/source/v2/tenant-routes.blade.md

```diff
@@ -7,7 +7,9 @@ section: content
 
 # Tenant Routes {#tenant-routes}
 
-Routes within `routes/tenant.php` will have the `web` middleware group and the `IntializeTenancy` middleware automatically applied on them. This middleware attempts to identify the tenant based on the current hostname. Once the tenant is identified, the database connection, cache, filesystem root paths and, optionally, Redis connection, will be switched.
+Routes within `routes/tenant.php` will have the `web` middleware group and the `IntializeTenancy` middleware automatically applied on them. 
+
+The `IntializeTenancy` middleware attempts to identify the tenant based on the current hostname. Once the tenant is identified, the database connection, cache, filesystem root paths and, optionally, Redis connection, will be switched.
```

Comment:
This was split into 2 paragraphs for easier readability of the slightly separate thoughts.